subsys/bluetooth/host/hci_core.h

```diff
@@ -29,6 +29,7 @@ enum {
 /* bt_dev flags: the flags defined here represent BT controller state */
 enum {
 	BT_DEV_ENABLE,
+	BT_DEV_DISABLE,
```

Comment:
Is this  necessary ? Couldn't enable / disable be controlled by one single bit ?

Reply:
Having a DISABLE bit prevents hci.close() reentry which is something required as this can involve complex procedure like communication with another core.

Reply:
:+1: